lib/modules/core/chabok.js

```diff
@@ -49,9 +49,40 @@ export class AdpPushClient {
 
     setDefaultTracker = (defaultTracker) => AdpNativeModule.setDefaultTracker(defaultTracker);
 
-    setUserInfo = (userInfo) => AdpNativeModule.setUserInfo(userInfo);
+    /**
+     * @deprecated the function has been replaced with AdpNativeModule.setUserAttributes()
+     */
+    setUserInfo = (userInfo) => AdpNativeModule.setUserAttributes(userInfo);
+
+    /**
+     * @deprecated the function has been replaced with AdpNativeModule.getUserAttributes()
+     */
+    getUserInfo = () => AdpNativeModule.getUserAttributes();
+
+
+    setUserAttributes = (userAttributes) => AdpNativeModule.setUserAttributes(userAttributes);
+    getUserAttributes = () => AdpNativeModule.getUserAttributes();
+
+    incrementUserAttribute = (attribute, value) => {
+        value = value || 1;

```

Comment:
check value with `isNaN` or `typeof === 'number'`

Reply:
even throw an exception for invalid values

Reply:
Done.